src/Redux/reducers/StylesReducer.js

```diff
@@ -0,0 +1,18 @@
+import { stylesActions } from "../actionCreators";
+
+const InitialState = {
+  hamburgerMenu: false,
+};
+
+const stylesReducer = (state = InitialState, action) => {
```

Comment:
Looks like this is maintaining the visibility state of hamburger menu. You could rename the reducer accordingly. The reducer could be called something like `mobileMenuReducer` or something similar. The state could be `visible` instead of `hamburgerMenu`. 

You will have to update the actions also accordingly.

Reply:
This reducer could be generic, where we can maintain the states required for the styles

Reply:
On a second thought the problem should be solved using media queries rather than using redux.